scripts/performance-helper.psm1

```diff
@@ -224,8 +224,15 @@ function Wait-ForRemote {
     param ($Job)
     # Ping sidechannel socket on 9999 to tell the app to die
     $Socket = New-Object System.Net.Sockets.UDPClient
-    $Socket.Send(@(1), 1, $RemoteAddress, 9999) | Out-Null
-    Wait-Job -Job $Job -Timeout 120 | Out-Null
+    for ($i = 0; $i -lt 120; $i++) {
+        $Socket.Send(@(1), 1, $RemoteAddress, 9999) | Out-Null
+        $Socket.Send(@(1), 1, $RemoteAddress, 9999) | Out-Null
```

Comment:
Why two? Seems unnecessary.

Reply:
I thought it was failing on the remote side, not the client side, so I just wanted to make sure they would send. I'll bring it back down to 1 now.